requirements.txt

```diff
@@ -1,5 +1,5 @@
 pyDeprecate>=0.3.2
-numpy>=1.20
+numpy>=1.20,<1.25
```

Comment:
Why the upper bound? Was there a breaking change in version 1.25?

Reply:
Yes, I think it was some interplay between the latest numpy and scipy